packages/fxa-admin-server/src/test/lib/resolvers/account-resolver.spec.ts

```diff
@@ -29,32 +30,45 @@ const EMAIL_BOUNCE_1 = randomEmailBounce(USER_1.email);
 const USER_2 = randomAccount();
 
 describe('accountResolver', () => {
+  // Used to get the shape of the return type
+  const contextDummy = (false as true) && mockContext();
```

Comment:
Does `let context: ReturnType<mockContext>;` work?

Reply:
Nope, I tried that one first before googling for this solution. It's because mockContext is a value, not a type..... and in typing that, I realized what *does* work!
```let context: ReturnType<typeof mockContext>;```